Dockerfile

```diff
@@ -38,6 +38,18 @@ RUN zypper addrepo https://download.opensuse.org/repositories/isv:SUSE:Edge:Edge
     nm-configurator && \
     zypper clean -a
 
+# Make adjustments for running guestfish and image modifications on aarch64
+# guestfish looks for very specific locations on the filesystem for UEFI firmware
+# and also expects the boot kernel to be a portable executable (PE), not ELF.
+RUN if [[ $(uname -m) == "aarch64" ]]; then \
+	zypper install -y qemu-uefi-aarch64; \
```

Comment:
How much space does this package take up? If it's trivial, it may make sense to go ahead and install the aarch64 emulator in the x86 image as well to make working on cross complications easier in the future.

Reply:
This is something we already do in the downstream builds.